(doc) add taskDefault example (#224)
kestra-io/kestra.io: content/docs/05.developer-guide/01.flow.md

|`taskDefaults.[].values.xxx`| The task property that you want to be set as default.                                                                                                                                        |
|`disabled`| Set it to `true` to disable execution of the flow.                                                                                                                                           |

### `taskDefaults`

> `taskDefaults` avoid to repeat task properties. For example:

Reviewer: ```suggestion
>You can add task defaults to avoid repeating task properties on multiple occurrences of the same task in a `taskDefaults` properties. For example:
```

I also prefer valid sentences in documentation, it's easier to read.

```yaml
id: python_test
namespace: test

tasks:

  - id: hello_from_docker

>     description: Parse each Ebay ad to retrieve interesting data.

Reviewer: ```suggestion
```

    type: io.kestra.core.tasks.scripts.Python
    inputFiles:
      main.py: |
        print("Hello World!")

  - id: hello_from_docker_2

>     description: Parse each Ebay ad to retrieve interesting data.

Reviewer: ```suggestion
```
